src/rgw/rgw_rados.cc

```diff
@@ -9542,6 +9542,11 @@ int RGWRados::set_attrs(void *ctx, const RGWBucketInfo& bucket_info, rgw_obj& ob
                         map<string, bufferlist>& attrs,
                         map<string, bufferlist>* rmattrs)
 {
+  rgw_obj obj = src_obj;
```

Comment:
this isn't compiling because `rgw_obj& obj` is a parameter. maybe change the param to `const rgw_obj& src_obj`?